megamek/i18n/megamek/client/messages.properties

```diff
@@ -747,6 +747,8 @@ CommonSettingsDialog.title=Client Settings
 CommonSettingsDialog.tooltipDelay=Tooltip popup delay:
 CommonSettingsDialog.tooltipDismissDelay=Tooltip dismiss delay:
 CommonSettingsDialog.tooltipDismissDelayTooltip:Sets the delay before a tooltip is dismised.  -1 uses the default for your look and feel.
+CommonSettingsDialog.tooltipDistSuppressionTooltip=<html>To prevent tooltips from following the cursor on the board, the tooltip will be supressed if the mouse moves more than this distance between updates. <br>The default value is the diagonal of a single</html>
```

Comment:
the diagonal of a single what?

Reply:
The answer is "hex", although that's not strictly accurate.  It's really the diagonal of the hex's bounding box, which is longer, but I don't know that the distinction is super important.  Although the diagonal of a hex is a curious concept.